src/verifications/dtos/verify-application-vcs-response.dto.ts

```
@@ -0,0 +1,11 @@
+export class VerifyApplicationVcsResponseDto {
+  applicationId: string;
+  status: 'verified' | 'partially_verified' | 'unverified';
+  message: string;
+  files: {
+    id: any;
+    filePath: string | null;
+    status: 'verified' | 'unverified';
+    message: string;
+  }[];
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)